tests/inlineasm/asmpushpop.py

```diff
@@ -5,5 +5,15 @@ def f(r0, r1, r2):
     pop({r0})
     pop({r1, r2})
 
+@micropython.asm_thumb
+def g():
+    b(START)
+    label(SUBROUTINE)
+    push({lr})  # push return address
+    mov(r0, 7)
+    pop({pc})   # return
+    label(START)
+    bl(SUBROUTINE)
+
 
-print(f(0, 1, 2))
+print(f(0, 1, 2), g())
```

Comment:
please write the new test as a separate print, ie:
```python
print(f(...))
print(g())
```

Reply:
Done. See new commit 3037b57e8b86d54f4c6ac8b4a6353bc1f025c391.